homeassistant/components/light/recorder.py

```diff
@@ -0,0 +1,12 @@
+"""Integration platform for recorder."""
+from __future__ import annotations
+
+from homeassistant.core import HomeAssistant, callback
+
+from . import ATTR_EFFECT_LIST, ATTR_SUPPORTED_COLOR_MODES
+
+
+@callback
+def exclude_attributes(hass: HomeAssistant) -> set[str]:
+    """Exclude static attributes from being recorded in the database."""
+    return {ATTR_SUPPORTED_COLOR_MODES, ATTR_EFFECT_LIST}
```

Comment:
We can add `ATTR_MIN_MIREDS` and `ATTR_MAX_MIREDS`

Reply:
Good catch 👍